### ISSUE TITLE
D-Wave results are being used as weight coefficients and are all are 1's

### ISSUE THREAD

**Issue by stephenlb** [dwave-examples/qboost#8]
# What is happening?

### TL;DR - This codebase demonstrates _no value_ in representing D-Wave Quantum for AI application advancements.

Running the examples and exploring the output weights. What is really happening?  
> It appears that nothing logically substantial is happening from D-Wave.

After running the examples, the D-Wave output weights are 1's `[1 1 1...]`. That means nothing is really happening here.  There's no value in this.  Repeated executions result in varied outcomes ( due to RNG initializations ).  However the D-wave final weights remain the same. This provides no additive value to the classic model for training AI.

The `readme.md` file states that: 

> This code demonstrates the use of the D-Wave system to solve a binary classification problem using the Qboost algorithm.

However there's no clear alignment with the results of the program output.

```shell
python demo.py --wisc
python demo.py --mnist
```

### Possible Considerations

There does not appear to be a mistake here.  However the value of D-wave is unclear.  We can consider the following:

1. This `demo.py` example isn't worthwhile and can't demonstrate D-wave/Quantum's value in AI/ML advancements.  Would another example work?
2. The code isn't demonstrating anything useful.  The training models are 100% trained classically and the data exchanged with D-Wave returns 1's `[1 1 1...]`.  This represents no value in the model.
3. Is it possible that the intent here is to demonstrate that the D-Wave QPU is able to verify the classically trained model.  If that is the case, then what value does this example bring to ML/AL advancements?

### Output Results Screenshot

![image](https://user-images.githubusercontent.com/45214/89694783-94cfec00-d8c6-11ea-8e08-ade1a87d0c60.png)

**Comment by stephenlb** [comment]
Coming to a possible conclusion. The DWaveSamper can be used to discover the most meaningful features to be used in an SVM.  While the qboost code example didn't make this clear, nor did it achieve meaningful results after repeated runs, the hybrid quantum and classical ML model allows optimization of the discovery for meaningful features.

**Comment by joelgdwave** [comment]
Hello @stephenlb, sorry for the delay. I would like to talk to you about your work, here, and what you're looking for. My name is Joel Gottlieb, and you can please reach me at jgottlieb@dwavesys.com.

**Comment by stephenlb** [comment]
Hi Joel,  thank you.  Email sent.

**Comment by nehcgnay** [comment]
Hello, I have the same question. Is there any updates or explanations can be added? Thanks

**Comment by stephenlb** [comment]
Hi!  I've eventually figured it out.  The examples only sort of make sense.  However I was able to complete my application with total success.
